service/storage_service.cc

```diff
@@ -2747,11 +2747,7 @@ future<> storage_service::on_alive(gms::inet_address endpoint, gms::endpoint_sta
     if (is_normal_token_owner) {
         co_await notify_up(endpoint);
     }
-    bool replacing_pending_ranges = _replacing_nodes_pending_ranges_updater.contains(endpoint);
-    if (replacing_pending_ranges) {
-        _replacing_nodes_pending_ranges_updater.erase(endpoint);
-    }
-
+    bool replacing_pending_ranges = false;
```

Comment:
What was it used for in the past?
What happened to this logic?

Reply:
Good.  It seemed vaguely familiar, so now ai know why :)

Anyhow, let's refer to it in the changelog message as part of the background story.

Otherwise, LGTM.

Reply:
Updated the PR top-level comment